introduction.md

```diff
@@ -1,4 +1,8 @@
 Blue Chill Labs
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?